Market Maker Guide.md

```diff
@@ -230,26 +228,25 @@ mainnet > listpeers -j
 
 ## Your First Trade
 
-On Simnet simply wait for about 15 minutes and you'll have channels and are read to go (check with `getinfo` and `getbalance`). On Testnet/Mainnet, start by deposit some coins: 
+On Simnet simply wait for about 15 minutes and you should see channels with balance (check with `getbalance`). On Testnet/Mainnet, start by depositing some funds into your xud node: 
```

Comment:
I'd shill it a bit more like `The deposit command is powered by Boltz and it will automatically open a balanced lightning channel to you.`

Reply:
Added Boltz shill